src/getters/index.ts

```diff
@@ -749,21 +749,22 @@ export function getSpotApy<
 
 /**
  * Sums amounts delegated to given operator by given wallet.
- * @param address Wallet address.
- * @param operator.delegators Collection of delegators.
- * @returns Wallet's delegation amount sum across all operators.
  */
-export function getDelegatedAmountForWallet<T extends Pick<ParsedOperator, 'delegators'>>(
+export function getDelegatedAmountForWallet(
     address: string,
-    { delegators }: T,
+    { delegators, exchangeRate }: ParsedOperator,
 ): BN {
     const addr = address.toLowerCase()
 
-    return delegators.reduce(
-        (sum, { delegator, amount }) =>
-            delegator.toLowerCase() === addr ? sum.plus(amount) : sum,
-        toBN(0),
-    )
+    return delegators
+        .reduce(
```

Comment:
This I'm not sure of. Waiting for @jtakalai to confirm (on Slack) if a single delegator can appear twice in the collection.

- If no, they we can `find` the one (skip the reduce).
- If yes, then what do we sum? `operatorTokenBalanceWei` sounds like it's already all they got.